src/components/reusable/radioButton/radioButtonGroup.ts

```diff
@@ -90,12 +90,24 @@ export class RadioButtonGroup extends FormMixin(LitElement) {
           : null}
 
         <div class="${this.horizontal ? 'horizontal' : ''}">
-          <slot></slot>
+          <slot @slotchange=${this._handleSlotChange}></slot>
         </div>
       </fieldset>
     `;
   }
 
+  private _handleSlotChange() {
+    this._updateChildren();
+    this.requestUpdate();
```

Comment:
Probably don't need to manually request update here. That's usually only necessary if you are using a `query` decorator somewhere in your template, because queries are updated post-render.

Reply:
Removed